Java/grades.java

```diff
@@ -0,0 +1,50 @@
+/*Java Program to demonstrate the use of If else-if ladder.
+• program of grading system for fail (&lt;50),
+• D grade -marks&gt;=50 &amp;&amp; marks&lt;60
+• C grade- marks&gt;=60 &amp;&amp; marks&lt;70,
+• B grade- marks&gt;=70 &amp;&amp; marks&lt;80 ,
+• A grade- marks&gt;=80 &amp;&amp; marks&lt;90
+• and A+ - marks&gt;=90 &amp;&amp; marks&lt;100.*/
+import java.util.Scanner;
+public class Grade
+{
+public static void main(String args[])
+{
+System.out.println(&quot;Enter Marks: &quot;);
+ABC S=new ABC();
+S.display();
```

Comment:
- Reformat the whole code with appropriate spaces
- comments which you wrote, in the beginning, should be placed at appropriate grade assigned

Reply:
done